packages/a11y/src/index.ts

```diff
@@ -21,52 +23,58 @@ function getWebpackConfig(p: string) {
     .getConfig();
 }
 
-function printResults(results: axe.AxeResults, impact: number): string {
+function formatResults(results: IResult[], impact: number): string {
   const msg: string[] = [];
   let index = 1;
-  results.violations.forEach((violation) => {
-    if (impactArray.indexOf(violation.impact) + 1 >= impact) {
-      violation.nodes.forEach((node) => {
-        msg.push(`${index++}. ${chalk.red(violation.id === 'duplicate-id' ? 'Document' : node.target[0])}: (Impact: ${violation.impact}) ${node.failureSummary}`);
+  results.forEach((res) => {
+    if (res.error) {
+      msg.push(`${index++}. ${res.comp}: Error while testing component - ${res.error}`);
+    } else if (res.result) {
+      res.result.violations.forEach((violation) => {
+        if (impactArray.indexOf(violation.impact) + 1 >= impact) {
+          violation.nodes.forEach((node) => {
+            const compName = (`${res.comp} - ${node.target[0]}`);
+            msg.push(`${index++}. ${chalk.red(compName)}: (Impact: ${violation.impact}) ${node.failureSummary}`);
+          });
+        }
       });
     }
   });
-  if (msg.length === 0) {
-    return 'No errors found';
-  } else {
-    return msg.join('\n\n');
-  }
+  return msg.join('\n\n');
 }
 
-export async function a11yTest(p: string, impact: number) {
+export async function a11yTest(entry: string | string[], impact: number) {
   let server: IServer | null = null;
   let browser: puppeteer.Browser | null = null;
-  console.log('Running accessibility tests...');
+  consoleLog('Running accessibility tests...');
```

Comment:
I think that by default our commands should print only errors. npm prints the name of the script it's running anyway, so this is redundant.